packages/eas-cli/src/graphql/queries/BuildQuery.ts

```diff
@@ -0,0 +1,75 @@
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../client';
+import { Build } from '../types/Build';
+
+type Filters = Partial<Pick<Build, 'platform' | 'status'>> & {
+  order?: number;
+  limit?: number;
+};
+
+type ArtifactFragmentType = Pick<Build, 'artifacts'>;
+type PlatformAndArtifactFragmentType = Pick<Build, 'platform' | 'artifacts'>;
```

Comment:
- I think the names are inaccurate (`artifact` -> `artifacts`) and the `Type` suffix seems weird.
- I don't have an idea for better names now - feel free to suggest sth.
- I'd just define the types inline if needed. You're not exporting them anyway.

Reply:
I've reduced those types to one, common for each query function, named: `BuildQueryResult` - IMO it's pretty accurate name. As the type is used in multiple places across the same file, I didn't inline it.